roles/nameserver/tasks/firewall.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Enable firewalld
```

Comment:
Is starting firewalld enough to take over from any other services, like a default iptables or whatever?